music_player/scripts/music_player_server.py

```diff
@@ -58,6 +59,8 @@ def __init__(self):
             outfile.close()
 
         self.file_names = [join(self.audio_folder, f[0]) for f in file_set]
+        if rospy.get_param("~shuffle", False):
+            random.shuffle(self.file_names)
```

Comment:
This will shuffle it once at the start, and then it always stays the same. Is this what we want? That would mean every walking group would have the same order of songs? Well, until we restart it...